app/main.py

```diff
@@ -1,34 +1,115 @@
-class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+from typing import List
 
-
-class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
-
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
-
-    def fire(self, row, column):
-        # Change the `is_alive` status of the deck
-        # And update the `is_drowned` value if it's needed
-        pass
+from app.history_shoots import history_shoots
+from app.ship import Ship
 
 
 class Battleship:
-    def __init__(self, ships):
+
+    def __init__(self, ships: List[tuple]):
         # Create a dict `self.field`.
         # Its keys are tuples - the coordinates of the non-empty cells,
         # A value for each cell is a reference to the ship
         # which is located in it
```

Comment:
If you want to describe the behavior of methods or classes, I would recommend using docstring. Then you need to change the description that the method should do something. He just does it.